.github/workflows/functional_test.yaml

```diff
@@ -63,8 +65,8 @@ jobs:
           curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl
           chmod +x kubectl
           sudo mv kubectl /usr/local/bin/
-          mkdir -p ${HOME}/.kube
-          touch ${HOME}/.kube/config
+          mkdir -p /tmp/kube-config-splunk-otel-collector-chart-splunk-integration-testing
+          touch /tmp/kube-config-splunk-otel-collector-chart-splunk-integration-testing/config
```

Comment:
I don't think this is needed. I believe KUBECONFIG is the filepath and not the dir path; having KUBECONFIG env in this step should be enough for minikube.

Reply:
yes, you are right, removed
Thank you